src/astro/databases/base.py

```diff
@@ -201,15 +201,44 @@ def append_table(
         self,
         source_table: Table,
         target_table: Table,
-        if_conflicts: AppendConflictStrategy = "exception",
```

Comment:
Why was this parameter removed?

Reply:
https://en.wikipedia.org/wiki/Merge_(SQL) - Yea Merge for most DB MySQL, Postres, SQLite this is Insert and do some actions on conflicts

Reply:
- https://www.postgresql.org/docs/current/sql-insert.html
- https://dev.mysql.com/doc/refman/8.0/en/insert.html

whereas Databricks & Snowflake do have a Megre function

- https://docs.databricks.com/delta/delta-update.html#upsert-into-a-table-using-merge
- https://docs.snowflake.com/en/sql-reference/sql/merge.html

Reply:
We agreed on new Conflict Strategy values and started using them.